tests/trilinos/trilinos_sparsity_pattern_04.cc

```diff
@@ -196,11 +196,6 @@ void test ()
 
 int main (int argc, char **argv)
 {
-  std::ofstream logfile("output");
-  deallog.attach(logfile);
-  deallog.depth_console(0);
-  deallog.threshold_double(1.e-10);
```

Comment:
This is a cleanup of #1231. The code contains the exact same lines further down, but enclosed for myid==0 which is the standard way for writing MPI test cases. At least I think so since that's what most other tests do where I copy from. We only collect output from processor 0, I don't know what happened before.